lib/upipe-hbrmt/upipe_pack10bit.c

```diff
@@ -112,32 +114,14 @@ static bool upipe_pack10bit_handle(struct upipe *upipe, struct uref *uref,
     struct upipe_pack10bit *upipe_pack10bit = upipe_pack10bit_from_upipe(upipe);
     const char *def;
     if (unlikely(ubase_check(uref_flow_get_def(uref, &def)))) {
-        upipe_pack10bit_store_flow_def(upipe, uref);
+        upipe_pack10bit_store_flow_def(upipe, NULL);
+        upipe_pack10bit_require_ubuf_mgr(upipe, flow_format);
         return true;
     }
 
     if (upipe_pack10bit->flow_def == NULL)
         return false;
 
-    size_t bytes;
-    uref_block_size(uref, &bytes);
-    if (!bytes || bytes % 32) {
-        upipe_err(upipe, "input buffer for pack10 must be padded to 32 bytes");
-        uref_free(uref);
-        return true;
-    }
-
-    if (unlikely(!upipe_pack10bit->ubuf_mgr)) {
```

Comment:
I think this check should stay with just a return false, because the ubuf_mgr has already been requested.